doc/src/user_guide.rst

```diff
@@ -1076,6 +1076,14 @@ Parameterised with
 
     ``blocking`` | ``non-blocking``
 
+4. ``region`` --- optional specification of a region to be written,
+   defined via either diametrically opposite vertices of a bounding
+   cuboid, or the name of a domain boundary (in the latter case the
+   solution in all elements that have faces on the domain boundary
+   is written):
```

Comment:
are written?

Reply:
'is' refers to solution (singular) taken from a bunch of elements, so I think correct?